spring-rabbit/src/test/java/org/springframework/amqp/rabbit/listener/ListenFromAutoDeleteQueueTests.java

```diff
@@ -141,6 +147,7 @@ public void testAutoDeclareFalse() throws Exception {
 		//Prevent a long 'passiveDeclare' process
 		BlockingQueueConsumer consumer = mock(BlockingQueueConsumer.class);
 		doThrow(RuntimeException.class).when(consumer).start();
+		when(consumer.getBackOffExecution()).thenReturn(mock(BackOffExecution.class));
```

Comment:
Cool! This empty mock returns `0` for the `backOffExecution.nextBackOff()` :smile: 
Merging...